assets/css/style.scss

```diff
@@ -247,18 +247,23 @@ Divs
 *******************************************************************************/
 
 .resource-row {
-  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
 }
-
 .resource-div {
-  width: 240px;
-  padding: 20px;
-  margin: 10px;
-  background-color: white;
-  display: inline-block;
-  text-align: center;
-  vertical-align: top;
-  box-shadow: 0px 0px 10px #888888;
+ display: flex;
+ flex-direction: column;
+ width: 44%;

```

Comment:
Re-set width to concrete size - this enables the "stacking" behavior as the width of the screen shrinks as opposed to the tiles shrinking into very tall, skinny columns.  I've found that 240px seems to be the sweet spot where the tiles are aesthetically pleasing and fit the content well.

Reply:
I personally like %'s for responsive web design & with different size devices in mind, but there's a downside with much wider screens I suppose.

Reply:
I agree with you that there are some benefits to using percentages.  In this case, with a % value here, when I shrink the screen width, the columns all become very skinny and end up with one word per line and some overflow.  Personally I think it's a better user experience to have them start stacking instead as the screen width gets shorter.

Are you observing a different behavior on your end with the percentages?